app/scripts/templates/settings.mustache

```diff
@@ -1,38 +1,26 @@
-<div id="main-content" class="card">
-  <header>
-    <h1 id="fxa-settings-header">{{#t}}Manage account{{/t}}</h1>
+<div id="fxa-settings-header-wrapper">
+  <header id="fxa-settings-header">
+    <h1 id="fxa-manage-account">{{#t}}Manage account{{/t}}</h1>
+    {{#showSignOut}}
+    <button id="signout" class="settings-button secondary">{{#t}}Sign out{{/t}}</button>
+    {{/showSignOut}}
   </header>
+  <div class="settings-success-wrapper">
+    <div class="success settings-success">Hello</div>
+  </div>
+</div>
 
-  <section>
-    <div class="error"></div>
-    <div class="success"></div>
-
-    <div class="avatar-wrapper avatar-view">
+<div id="fxa-settings-profile-header">
+    <div class="avatar-wrapper avatar-settings-view">
     </div>
-    <p class="change-avatar-text links">
-      <a href="/settings/avatar/change">{{#t}}Change{{/t}}</a>
+    <p class="signed-in-email-message">
+      {{#t}}Hello, <span class="username">%(username)s</span>{{/t}}
```

Comment:
nit: to make it easier to translate we can change to `{{#t}}Hello, <span>%(username)s</span>{{/t}}`
and we can target CSS with `signed-in-email-message span`